Source/SwiftLintFramework/Rules/OpeningBraceRule.swift

```diff
@@ -25,16 +36,89 @@ public struct OpeningBraceRule: Rule {
             "func abc()\n\t{ }",
             "[].map(){ $0 }",
             "[].map( { } )"
+        ],
+        corrections: [
+            "struct Rule{}\n": "struct Rule {}\n",
+            "struct Rule\n{\n}\n": "struct Rule {\n}\n",
+            "struct Rule\n\n\t{\n}\n": "struct Rule {\n}\n",
+            "struct Parent {\n\tstruct Child\n\t{\n\t\tlet foo: Int\n\t}\n}\n":
+                "struct Parent {\n\tstruct Child {\n\t\tlet foo: Int\n\t}\n}\n",
+            "[].map(){ $0 }\n": "[].map() { $0 }\n",
+            "[].map( { } )\n": "[].map({ } )\n",
```

Comment:
Maybe I can add example `"[].map( { })\n": "[].map({ })\n"` too as a contrast.
In this case, we rather *want* to autocorrect :grin:.

I personally think we should autocorrect for `"[].map( { } )\n": "[].map({ } )\n"` as well,
and provide companion `TrailingBraceRule.swift` later.